ui/src/tool-panel/location-track/location-track-geometry-infobox.tsx

```diff
@@ -38,13 +38,14 @@ export const LocationTrackGeometryInfobox: React.FC<LocationTrackGeometryInfobox
     const { t } = useTranslation();
     const [useBoundingBox, setUseBoundingBox] = React.useState(true);
     const viewportDep = useBoundingBox && viewport;
-    const [sections, elementFetchStatus] = useLoaderWithStatus(
+    const [sections, elementFetchStatus] = useRateLimitedLoaderWithStatus(
```

Comment:
Meillä alkaa olla aika monta näitä eri loader-variantteja, kun aiemmasta oli jo perus-`useLoader`, `useLoaderWithStatus`, `useLoaderWithTimer` ja `useOptionalLoader`. Mietin että olisikohan järkeä alkaa yhdistelemään näitä jotenkin yhden katon (tai ainakin nykyistä pienemmän määrän kattoja) alle... Jos näistä koskaan halutaan yhdistellä ominaisuusjoukkoja (kuten esim. tässä rate limit ja statuksen palautus,) niin ka. ominaisuusjoukolle saattaa nykyisellään joutua tekemään taas kokonaan uuden `useLoader`-variantin. Ei toki suoraan tämän tiketin asiaa, mutta jossain yhteydessä näitä voisi mielestäni miettiä.